setup.py

```diff
@@ -41,8 +41,8 @@ def local_read(filename):
     keywords='lsst',
     packages=find_packages(exclude=['docs', 'tests*']),
     install_requires=[
-        'sqre-apikit==0.0.10',
-        'requests==2.13.0',
+        'sqre-apikit==0.1.1',
+        'requests>=2.13.0,<3.0.0',
```

Comment:
This is a great change. Much easier to manage dependencies with a range. As long as the API is stable between major versions.